From fd7132a5d7e17762158b519cc8bbc87110155c7a Mon Sep 17 00:00:00 2001
From: Tiejun Chen <tiejun.chen@windriver.com>
Date: Fri, 2 Dec 2011 10:48:24 +0800
Subject: [PATCH 10/35] powerpc/p204xrdb: correct sdhci node description

Per current sdhci driver to change 'sdhci,auto-cmd12' to
'fsl,sdhci-auto-cmd12', this make mmc work well.

Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 arch/powerpc/boot/dts/p2040rdb-usdpaa.dts |    2 +-
 arch/powerpc/boot/dts/p2040rdb.dts        |    2 +-
 arch/powerpc/boot/dts/p2041rdb-usdpaa.dts |    2 +-
 arch/powerpc/boot/dts/p2041rdb.dts        |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/powerpc/boot/dts/p2040rdb-usdpaa.dts b/arch/powerpc/boot/dts/p2040rdb-usdpaa.dts
index 590da5b..61e3e8a 100644
--- a/arch/powerpc/boot/dts/p2040rdb-usdpaa.dts
+++ b/arch/powerpc/boot/dts/p2040rdb-usdpaa.dts
@@ -714,7 +714,7 @@
 			compatible = "fsl,p2040-esdhc", "fsl,esdhc";
 			reg = <0x114000 0x1000>;
 			interrupts = <48 2 0 0>;
-			sdhci,auto-cmd12;
+			fsl,sdhci-auto-cmd12;
 			clock-frequency = <0>;
 		};
 
diff --git a/arch/powerpc/boot/dts/p2040rdb.dts b/arch/powerpc/boot/dts/p2040rdb.dts
index a6b7591..92e9b19 100644
--- a/arch/powerpc/boot/dts/p2040rdb.dts
+++ b/arch/powerpc/boot/dts/p2040rdb.dts
@@ -705,7 +705,7 @@
 			compatible = "fsl,p2040-esdhc", "fsl,esdhc";
 			reg = <0x114000 0x1000>;
 			interrupts = <48 2 0 0>;
-			sdhci,auto-cmd12;
+			fsl,sdhci-auto-cmd12;
 			clock-frequency = <0>;
 		};
 
diff --git a/arch/powerpc/boot/dts/p2041rdb-usdpaa.dts b/arch/powerpc/boot/dts/p2041rdb-usdpaa.dts
index 60bc422..dfe5001 100644
--- a/arch/powerpc/boot/dts/p2041rdb-usdpaa.dts
+++ b/arch/powerpc/boot/dts/p2041rdb-usdpaa.dts
@@ -716,7 +716,7 @@
 			compatible = "fsl,p2041-esdhc", "fsl,esdhc";
 			reg = <0x114000 0x1000>;
 			interrupts = <48 2 0 0>;
-			sdhci,auto-cmd12;
+			fsl,sdhci-auto-cmd12;
 			clock-frequency = <0>;
 		};
 
diff --git a/arch/powerpc/boot/dts/p2041rdb.dts b/arch/powerpc/boot/dts/p2041rdb.dts
index b2e2e74..313b5ec 100644
--- a/arch/powerpc/boot/dts/p2041rdb.dts
+++ b/arch/powerpc/boot/dts/p2041rdb.dts
@@ -707,7 +707,7 @@
 			compatible = "fsl,p2041-esdhc", "fsl,esdhc";
 			reg = <0x114000 0x1000>;
 			interrupts = <48 2 0 0>;
-			sdhci,auto-cmd12;
+			fsl,sdhci-auto-cmd12;
 			clock-frequency = <0>;
 		};
 
-- 
1.7.9.7

